atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -139,19 +139,13 @@ private void cas(Client client, Long oldVal, long newVal) {
                     ConsistencyLevel.EACH_QUORUM);
         } catch (Exception e) {
             String msg = String.format("[CAS] Error trying to set from %s to %s", oldVal, newVal);
```

Comment:
We can either remove the `msg` variable or keep just the template and do the merging in of values when doing the logging